lib/clusters/ballastConfiguration.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+
+const { Cluster, ZCLDataTypes } = require('zigbee-clusters');
+
+const COMMANDS = {}
+
+const ATTRIBUTES = {
+
+  /* 0x0000: PhysicalMinLevel (optional, read-only)
+   * Description:   Minimum light level the ballast can achieve.
+   * Default:       0x01
+   * Range:         0x01 – 0xfe
+   */
+  PhysicalMinLevel: {
```

Comment:
The convention in this lib is `camelCase`, it seems you are using different types of casing in this file.